shared/wallets/transaction/index.tsx

```diff
@@ -309,24 +301,24 @@ const roleToColor = (role: Types.Role): string => {
   }
 }
 
-const getAmount = (role: Types.Role, amountXLM: string): string => {
+const getAmount = (role: Types.Role, amount: string, sourceAmount?: string): string => {
   switch (role) {
     case 'senderOnly':
-      return `- ${amountXLM}`
+      return `- ${sourceAmount || amount}`
     case 'airdrop':
     case 'receiverOnly':
-      return `+ ${amountXLM}`
+      return `+ ${amount}`
     case 'senderAndReceiver':
       return '0 XLM'
     default:
       throw new Error(`Unexpected role ${role}`)
   }
 }
 
-const AmountXLM = (props: AmountXLMProps) => {
+const Amount = (props: AmountProps) => {
```

Comment:
Since we don't display just XLM amounts, it made sense to tweak the name of this component.